src/sentry/api/endpoints/group_hashes.py

```diff
@@ -32,15 +41,30 @@ def get(self, request, group):
         :auth: required
         """
 
-        queryset = GroupHash.objects.filter(
-            group=group.id,
+        aggregations = [
+            ('argMax(event_id, timestamp)', None, 'event_id')
+        ]
+
+        filter_keys = {
+            'project_id': [group.project_id],
+            'group_id': [group.id]
+        }
```

Comment:
Is there a reason to define these (`aggregations`, `filter_keys`) outside of the `query` partial?